### PR TITLE
Add AZP configuration for CI

### PR DESCRIPTION
**Pull request by bkabrda**

### CODE REVIEW

File: .azure-pipelines/all.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- job: IntegrationTests
+  pool:
+    vmImage: "Ubuntu-16.04"
+  container:
+    image: golang:latest
+  steps:
+    - script: make testacc
+      displayName: Run integration tests
+      env:
+        CI: "true"
```
Comment (nmuesch) [inline .azure-pipelines/all.yml:22]: Do we also need RECORD=true or does this handle that?

Comment (bkabrda) [reply .azure-pipelines/all.yml:22]: Using `make testacc` actually does the right thing automatically. It runs tests against the real API without re-recording cassettes.

Comment (nmuesch) [reply .azure-pipelines/all.yml:22]: Ahh ok thanks!

**Comment by bkabrda** [comment]
The test suite seems to be pretty flaky, but working on that is not the aim of this PR. I'm going to merge and work on the flakiness next. Thanks for the review!

**Pull request merged**